metabase/metabase
# ⚠️ ⚠️ ⚠️  WIP: Embedding SDK PoC ⚠️ ⚠️ ⚠️

Relates to https://github.com/metabase/metabase/issues/33854

### Description

This adds a sample app with Embedding SDK PoC

### How to verify

Build a package:
1. `yarn build-embedding-sdk`

Then start regular metabase instance:
1. From root: `yarn dev`


Then start a sample host app:
1. From root: `cd host-app-example && yarn --force`.  (`--force` is needed to make sure embedding-sdk package is up-to-date after build)
2. `yarn start`
3. Update `host-app-example/src/App.js` to have your valid `question id`
4. Open `http://localhost:3004/`
